src/js/get-fundraiser.js

```diff
@@ -0,0 +1,62 @@
+const mapping = {
+  14274: { // France
+    url: ' https://www.helloasso.com/associations/rockclimber/formulaires/1/widget'
+  },
+
+  14474: { // Scotland
+    url: 'https://www.7amax.co.uk/'
+  },
+
+  14151: { // Malta
+    url: 'https://maltaclimbingclub.org/bolt-fund/'
+  },
+
+  14125: { // Philippines
+    url: 'http://www.climbphilippines.com/boltfund.html'
+  }
+};
+
+const intersection = function(array1, array2) {
+  return array1.filter(value => array2.includes(value));
+};
+
+export default function(document) {
+  // Only areas, waypoints, routes
+  if (!['r', 'w', 'a'].includes(document.type)) {
+    return null;
+  }
+
+  if (document.type === 'r') {
+    // for routes, only mountain_climbing and rock_climbing activity
+    if (intersection(document.activities, ['mountain_climbing', 'rock_climbing']).length === 0) {
+      return null;
+    }
+
+    // for routes, only P1 and P1+ routes
+    if (!['P1', 'P1+'].includes(document.equipment_rating)) {
+      return null;
+    }
+  }
+
+  // for waypoints, only climbing outdoor sites
+  if (document.type === 'w' && document.waypoint_type !== 'climbing_outdoor') {
+    return null;
+  }
+
+  const areas = [];
+  if (document.type === 'a') {
+    areas.push(document);
+  } else {
+    areas.push(...document.areas.filter((area) => area.area_type === 'range'));
```

Comment:
`areas.push(...document.areas.filter((area) => ['range', 'admin_limits', 'country'].includes(area.area_type));`

Reply:
Nope, the point of these three lines is to have first ranges, then admins_limit, then countries.

The logic behind is to display the item attached to the smallest entity. I also plan to include waypoints, and even routes in that logic : 

```
If an item is attached to the route => display it
else if an item is attached to it's main waypoint  => display it
else if an item is attached to an area of type range => display it
else if an item is attached to an area of type admin_limit => display it
else if an item is attached to an area of type country => display it
```